Source/Core/VideoBackends/D3D/PSTextureEncoder.cpp

```diff
@@ -147,12 +147,15 @@ size_t PSTextureEncoder::Encode(u8* dst, unsigned int dstFormat,
 
 		D3D::context->OMSetRenderTargets(1, &m_outRTV, nullptr);
 
-		ID3D11ShaderResourceView* pEFB = (srcFormat == PEControl::Z24) ?
-			FramebufferManager::GetEFBDepthTexture()->GetSRV() :
+		D3DTexture2D* efbTexture = (srcFormat == PEControl::Z24) ?
+			FramebufferManager::GetEFBDepthTexture() :
```

Comment:
I think I get it, the depth buffer should have been resolved here. Try switching to the master branch without any of the changes in the PR and then replace this function call by `FramebufferManager::GetResolvedEFBDepthTexture()`. That should fix it I think.